cai2hcl/converters/google/resources/project.go

```diff
@@ -1,10 +1,11 @@
-package cai2hcl
+package resources
```

Comment:
why project is not in a separate folder like compute?

Reply:
This file combines 2 APIs: cloudbilling and cloudresourcemanager. I am not sure where to put it

Reply:
I feel it's safe to put cloudbilling into the compute package, in my memory, cloud billing can only live with the project object.

Reply:
You mean into the cloudresourcemanager package?

Reply:
these should go in a folder called `resourcemanager`, at least for now - this billing info is part of the google_project resource, not a separate resource.

Reply:
Done